db/schema.rb

```diff
@@ -83,7 +83,6 @@
     t.integer  "signed_off_by"
     t.datetime "signed_off_at"
     t.integer  "project_id"
-    t.boolean  "mentor_fav"
```

Comment:
If I pull the changes from master and migrate the DB it's there - that's what i wanted to say 😉